garrysmod/gamemodes/terrortown/gamemode/corpse.lua

```diff
@@ -189,6 +206,11 @@ function CORPSE.ShowSearch(ply, rag, covert, long_range)
    local words = rag.last_words or ""
    local hshot = rag.was_headshot or false
    local dtime = rag.time or 0
+   
+   -- opportunity for addons to introduce ways for traitors(ideally) to interrupt corpse searching.
+   if hook.Run("TTTInterruptCorpseSearch", ply, rag, traitor) then
+      return
+   end

```

Comment:
I was going to leave this as is, so that modders are less inclined to override this default behavior of preventing id/search for burning corpses - are you sure you'd prefer this to be in the GM hook func?